### PR TITLE
Multi-tile airlock assemblies/pathing [NO GBP]

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Small fixes after https://github.com/tgstation/tgstation/pull/76732
- Creates base large subtype for mapping
- Door assembly naming consistent with other airlocks
- Base subtype large airlock no longer spawns in a normal sized airlock
- Renames station2 subtype to public for consistency

## Changelog

:cl: LT3
fix: Base large airlock subtype no longer spawns a normal sized airlock
image: Added mapping icons for large airlocks
code: Repathed all public airlocks to consistent naming
/:cl:

### CODE REVIEW

File: tools/UpdatePaths/76732_multi-tile_airlocks.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#comment Repathing for new multi-tile airlocks.
+
+/obj/machinery/door/airlock/multi_tile/glass : /obj/machinery/door/airlock/multi_tile/public/glass{@OLD}
+/obj/machinery/door/airlock/glass_large : /obj/machinery/door/airlock/multi_tile/public/glass{@OLD}
```
Comment (Maurukas) [inline tools/UpdatePaths/76732_multi-tile_airlocks.txt:4]: This file seems to be a duplicate. It shouldn't be in this folder, and there's a near identical copy in the Scripts folder

**Review by Maurukas — CHANGES_REQUESTED** [review]

File: tools/UpdatePaths/76732_multi-tile_airlocks.txt (hunk 1 above)
Comment (lessthnthree) [reply tools/UpdatePaths/76732_multi-tile_airlocks.txt:4]: Oop yeah, I forgot to delete it after running updatepaths.

**Comment by Wallemations** [comment]
Why does this have a sprite tag when icondiffbot says nothing changed?

**Comment by GoldenAlpharex** [comment]
Something that I also noticed was that it seems like the large airlocks still have a holder object to the bottom-right when the multi-tile airlock is "vertical" rather than horizontal. Would be cool if it could be fixed in this one too.

**Comment by lessthnthree** [comment]
> Something that I also noticed was that it seems like the large airlocks still have a holder object to the bottom-right when the multi-tile airlock is "vertical" rather than horizontal. Would be cool if it could be fixed in this one too.

TG doesn't have vertical large airlocks.

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -2095,12 +2095,12 @@
 	glass = TRUE
 //////////////////////////////////
 /*
-	Station2 Airlocks
+	Public Airlocks
```
Comment (Jacquerel) [inline code/game/machinery/doors/airlock.dm:2098]: If you're touching these please just replace them with 
`// Public Airlocks`

The "fancy" header fucks up our jsdocs and also takes up space for little benefit.

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/airlock.dm (hunk 2 above)
Comment (lessthnthree) [reply code/game/machinery/doors/airlock.dm:2098]: Sure

**Pull request merged**